docs/io/fits/index.rst

```diff
@@ -11,7 +11,8 @@ Introduction
 
 The :mod:`astropy.io.fits` package provides access to FITS files. FITS
 (Flexible Image Transport System) is a portable file standard widely used in
-the astronomy community to store images and tables.
+the astronomy community to store images and tables. This subpackage was
+originally developed as PyFITS.
```

Comment:
For consistency with the rest?
```suggestion
originally developed as ``PyFITS``.
```

Reply:
If you want double backticks, it should be `pyfits` because the actual package name is all lowercase.

Reply:
@taldcroft , so do you want to keep PyFITS or change to `pyfits` (all lowercase but with backticks)?

Reply:
I think not having backticks is fine here